docs/gettingstarted/quickstart.ipynb

```diff
@@ -2,24 +2,175 @@
  "cells": [
```

Comment:
<p>Would it be possible to import the full catalog, but then do that trick that you just implemented, by having a small box?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes, I added a pointer to the full Gaia catalog and read from a small cone around the objects of interest!